li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaFederatedConsumerImpl.java

```diff
@@ -552,17 +589,48 @@ synchronized public long position(TopicPartition partition, Duration timeout) {
 
   @Override
   synchronized public OffsetAndMetadata committed(TopicPartition partition) {
-    throw new UnsupportedOperationException("Not implemented yet");
+
+    return committed(partition, _defaultApiTimeout);
   }
 
   @Override
   synchronized public OffsetAndMetadata committed(TopicPartition partition, Duration timeout) {
```

Comment:
does vanilla block here for non-existent TPs? if so, we should probably document this